.github/workflows/release.yaml

```diff
@@ -20,7 +20,7 @@ jobs:
       - name: Set up Go
         uses: actions/setup-go@v4
         with:
-          go-version: 1.19
+          go-version: '1.20'
```

Comment:
Is this sufficient? In cnquery and cnspec, we have `>=1.20.4`.

https://github.com/mondoohq/cnquery/pull/1258/files

Reply:
yes that pulls the latest go 1.20 version